Week3/Homework-Week3/assets/morejs_11.js

```diff
@@ -0,0 +1,26 @@
+{
+  /* What if you add one more vehicle to the list, 
+  can you have that added to the advertisement without 
+  changing the code for question 10?*/
+
+  // Yes, this function allows to add more elements to the list
+  //and will console.log accordingly:
+
+  const vehicles = [' motorbike', ' scooter', ' caravan', ' bike'];
+
+  function advert() {
+    let advString = '';
+
+    for (let i = 0; i < vehicles.length; i++) {
+      if (i === vehicles.length - 1) {
+        advString += vehicles[i] + 's.';
+      } else {
+        advString += vehicles[i] + 's,';
+      }
```

Comment:
Ah, I forgot to check this one. It looks good but I think now it will result in:

`Amazing Joe's Garage, we service for motorbikes, scooters, caravans, and bikes`.

But the `,` before the `and` is not correct:) can you make it so that the `,` is not added before the `and`?